global/ramda.d.ts

```diff
@@ -1233,11 +1236,11 @@ declare namespace R {
         // propEq<T>(name: string, val: T, obj: {[index:string]: T}): boolean;
         // propEq<T>(name: string, val: T, obj: {[index:number]: T}): boolean;
         propEq<T>(name: string, val: T, obj: any): boolean;
-        // propEq<T>(name: number, val: T, obj: any): boolean;
+        propEq<T>(name: number, val: T, obj: any): boolean;
```

Comment:
Well, because it works and it's really useful. 😄 I'll change it, especially now that I know about `nth()`.